pydarn/plotting/maps.py

```diff
@@ -64,12 +66,14 @@ def plot_mapdata(cls, dmap_data: List[dict], ax=None,
                      parameter: Enum = MapParams.FITTED_VELOCITY,
                      record: int = 0, start_time: dt.datetime = None,
                      time_delta: float = 1,  alpha: float = 1.0,
-                     len_factor: float = 150, cmap: str = None,
-                     colorbar: bool = True, colorbar_label: str = '',
-                     title: str = '', zmin: float = None, zmax: float = None,
+                     len_factor: float = 150, color_vectors: bool = True,
+                     cmap: str = None, colorbar: bool = True,
+                     colorbar_label: str = '', title: str = '',
+                     zmin: float = None, zmax: float = None,
                      hmb: bool = True, boundary: bool = False,
                      radar_location: bool = False, map_info: bool = True,
-                     imf_dial: bool = True, **kwargs):
+                     imf_dial: bool = True, reference_vector: bool = False,
```

Comment:
In my opinion the reference vector should be on by default, even if the vectors are coloured.

Additionally, it would be good if the magnitude of the reference vector itself was an option. Instead of `True` or `False`, `reference_vector: int = 500`. This way it will just always default to 500, and the user can input `False` if they don't want it.

Reply:
Sounds good, I've made it so that the vector is on by default, and is controlled by entering an integer as `reference_vector = 1000` to set the size of the vector. It is on and 500 m/s as default, and can be turned off by setting the reference_vector to 0 or False.

The following code makes the three following plots (a:100m/s, b:off, c:default):
```python
map_file = "/Users/carley/Documents/data/maps/20150310.n.map"
map_data = pydarn.SuperDARNRead().read_dmap(map_file)

# Default default (radar fov turned off)
pydarn.Maps.plot_mapdata(map_data,record=360, 
            parameter=pydarn.MapParams.FITTED_VELOCITY,
            lowlat=50, color_vectors=True, reference_vector=1000)
plt.show()

pydarn.Maps.plot_mapdata(map_data,record=360, 
            parameter=pydarn.MapParams.FITTED_VELOCITY,
            lowlat=50, color_vectors=False,
            contour_fill=True, reference_vector=False)
plt.show()

pydarn.Maps.plot_mapdata(map_data,record=360, 
            parameter=pydarn.MapParams.FITTED_VELOCITY,
            lowlat=50, color_vectors=False,
            contour_fill=True)
plt.show()
```
<img width="586" alt="Screen Shot 2022-06-17 at 2 58 21 PM" src="https://user-images.githubusercontent.com/60905856/174400531-bc4ece55-8fdb-4467-a330-6c1c0a835153.png">
<img width="548" alt="Screen Shot 2022-06-17 at 2 58 52 PM" src="https://user-images.githubusercontent.com/60905856/174400537-5070dafd-1a49-45a1-8dba-af64d457d0bd.png">
<img width="556" alt="Screen Shot 2022-06-17 at 2 59 20 PM" src="https://user-images.githubusercontent.com/60905856/174400539-16ae72fa-174b-4636-a589-3870b9955484.png">